CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -880,25 +963,34 @@ Duration: 			Permanent
 Health:				50
 Size:				1m x 1m x 3m
 
-    		You assemble an active countermeasure system that eliminates 
-incoming explosive ordinance. When active, this system destroys any incoming 
-grenades, missiles, rockets or thrown weapons that it has line of sight to from 
-the top of its 3m mast within a 6m radius. Any incoming munitions are destroyed 
-before they can detonate. Cannot stop incoming bullets, lasers, or plasma. Will 
-stop some hostile Nanite attacks, see GM for specifics.
+    You assemble an active countermeasure system that eliminates incoming 
+explosive ordinance. When active, this system destroys any incoming grenades, 
+missiles, rockets or thrown weapons that it has line of sight to from the top of 
+its 3m mast within a 6m radius. Any incoming munitions are destroyed before they 
+can detonate. Cannot stop incoming bullets, lasers, or plasma. Will stop some 
+hostile Nanite attacks, see GM for specifics.
```

Comment:
Hostile nanite attacks (besides those delivered by Doctors via needle) are no longer in aesthetic, iirc.

Reply:
That aesthetic shift hasn't finished shipping yet. This will continue to be updated as the language elsewhere changes as well. Recall that this is work from almost a year ago getting propagated along.